packages/message-box/message-box.scss

```diff
@@ -34,20 +31,40 @@ $message-width: rem(560px);
         max-width: 100%;
     }
 
-    &--info:before {
-        background-color: $info;
+    &--info {
+        border: 1px solid $info;
+        background-color: change-color($info, $lightness: 97%);
+        &--dark {
+            border: 1px solid $info--darkbg;
+            background-color: change-color($info--darkbg, $lightness: 5%);
+        }
```

Comment:
Som nevnt over, her får du feil BEM syntaks: `jkl-message-box--info--dark`. I stedet for å definere dark-versjon inne i hver type kan du gjøre det på denne måten:

```scss
&--dark {
    &.jkl-message-box--info {
        border: 1px solid $info--darkbg;
        background-color: change-color($info--darkbg, $lightness: 5%);
    }
    &.jkl-message-box--warning {
        // ...
    }
    /* other styles */
}
```
Det gjør det også lettere å implementere en global dark mode senere, siden alle endringene er på ett sted 😊

Reply:
Done